contracts/aggregator/common/helpers.sol

```diff
@@ -0,0 +1,131 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract HelpersCommon {
+    using SafeERC20 for IERC20;
+
+    /**
+     * @dev Delegate the calls to Connector.
+     * @param _target Connector address
+     * @param _data CallData of function.
+    */
+    function spell(address _target, bytes memory _data) internal returns (bytes memory response) {
+        require(_target != address(0), "target-invalid");
+        assembly {
```

Comment:
Maybe use open zepp `address` lib for delegate call?
[This](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/83277ff916ac4f58fec072b8f28a252c1245c2f1/contracts/utils/Address.sol#L174) function

Reply:
Done!